Modules-Index.md

```diff
@@ -0,0 +1,91 @@
+## Hi there 👋
+
+### Documentation: [LLazyEmail Documentation](https://llazyemail.github.io/documentation/)
+
+<!--
```

Comment:
We can remove hidden Block, its not needed here

Reply:
https://github.com/LLazyEmail/documentation/pull/163